flake.nix

```diff
@@ -106,7 +107,9 @@
                 pkgs.darwin.apple_sdk.frameworks.Security
                 pkgs.darwin.apple_sdk.frameworks.SystemConfiguration
                 pkgs.darwin.apple_sdk.frameworks.CoreFoundation
-              ];
+              ] ++ (with inputs'.holonix.packages; [
+                hc
```

Comment:
Already included in this set above I think, included again here?

Reply:
yep, didn't remove here when added above when fixing ci.